src/App.js

```diff
@@ -1,26 +1,110 @@
 import React from 'react';
 import './App.scss';
+import { getTodos, getUser } from './api';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 class App extends React.Component {
   state = {
+    todosFromServer: [],
     todos: [],
     selectedUserId: 0,
+    selectedUser: {},
   };
 
+  componentDidMount() {
+    getTodos().then((list) => {
+      const initialTodos = list.data.filter(todo => todo.title);
+
+      this.setState({
+        todosFromServer: initialTodos,
+        todos: initialTodos,
+      });
+    });
+  }
+
+  setUser = (userId) => {
```

Comment:
what about componentDidUpdate for CurrentUser component?

Reply:
just realized how to implement it, I've updated the CurrentUser component